build-push-image/build-push-core/action.yml

```diff
@@ -71,17 +71,7 @@ runs:
         labels: ${{ steps.metadata.outputs.labels }}
         build-args: |
           ${{ steps.setup-build.outputs.build_args }}
-        cache-from: type=gha
```

Comment:
If I remove image cache, if I rerun the build, won't it be slow since it wasn't cache?